lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -35,22 +35,22 @@ sub edit_user : Path('/admin/user/edit') Args(1) RequireAuth HiddenOnSlaves
     my $form2 = $c->form(
         form => 'User::EditProfile',
         item => {
-            email            => $user->email,
+            username          => $user->name,
+            email             => $user->email,
             skip_verification => 0,
-            website            => $user->website,
-            biography        => $user->biography
+            website           => $user->website,
+            biography         => $user->biography
         },
     );
 
     if ($c->form_posted) {
-        if ($form->submitted_and_valid($c->req->params )) {
+        if ($form->submitted_and_valid($c->req->params) 
+            && $form2->submitted_and_valid($c->req->params )) {
```

Comment:
That sounds correct, i.e. they should both be submitted at the same time. It looks like it was previously the case that you'd get the redirect and "User successfully edited" banner even if one of the two forms contained errors.